examples/notebooks/load-modelling.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load Shedding, Shifting and Elastic Demand"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook provides a simple illustration of how load shedding, load shifting, and elastic demand can be modeled in PyPSA. Through the example, we'll simulate a single day of electrical consumption in a network with solar generation, which could represent a household or a small community.\n",
```

Comment:
To me, a second phrase hasn't sounded very clear at first reading: " a network with solar generation, which could represent...". Would it be possible to re-formulate a bit? Like "solar-powered network, which could represent..."?

Reply:
Edited to "we'll simulate a single day of electrical consumption in a solar-powered network, which could represent a household or a small community." ✔️

Reply:
Super! :)